src/stores/useHoloStore.js

```diff
@@ -0,0 +1,105 @@
+import { defineStore } from 'pinia'
+import WebSdk from '@holo-host/web-sdk'
+import useIsLoadingStore from './useIsLoadingStore'
+import useSignalStore from './useSignalStore'
+
+let client
+
+const makeUseHoloStore = ({ connectionArgs, MockWebSdk }) => defineStore('holo', {
+  state: () => ({
+    agentState: {},
+    happId: null,
+    connectionError: null,
+    is_auth_form_open: false,
```

Comment:
I've updated it to camel case because otherwise it's a mess right now. I'd like to open up a convo about this in the ui guild, have added it to the topics list.

Headline: I think snake case is much more readable than camel case, and it also makes talking to rust a little easier as JS representations of rust structs are camel case by default. What I've been playing with is having non function variables be snake case and functions be camel case. I realize this is slightly bizarre, not common practice at all, and probably not worth it, **but** it does make the code prettier and easier to read imho.

Anyway, for now I've put it to camelCase, pending a guild convo